stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/repository/RepositoryHelper.kt

```diff
@@ -257,4 +236,23 @@ internal class RepositoryHelper(
     internal suspend fun insertSyncState(newSyncState: SyncState) {
         syncStateRepository.insert(newSyncState)
     }
+
+    internal companion object {
+        fun create(factory: RepositoryFactory, scope: CoroutineScope): RepositoryHelper {
+            val userRepository = factory.createUserRepository()
+            val getUser: suspend (userId: String) -> User = { userId ->
+                userRepository.selectUser(userId) ?: error("User with the userId: `$userId` has not been found")
```

Comment:
This looks exactly the same as `selectUser`, but if there is a sense of reusing it somehow

Reply:
Not exactly. SelectUser is a member method of `RepositoryHelper`. And this place is a factory method for creating `RepositoryHelper`. This code will be separated from RepositoryHelper and in later PRs I'll move it to some `RepositoryHelperBuilder`